skycoin/skywire
# new/update flags on skywire-cli

Did you run `make format && make check`?

Fixes #1046 #1050 #1054 #1061

 Changes:	
- added `--public-rpc` flag to change CLIAddr from `localhost:3435` to `:3435`.
- added comparing step on adding hypervisors keys on `--hypervisor-pks <visor-pk>` flag, that if key equal to visor PK, then change this visor to hypervisor.
- added `--vpn-server-enable` flag to make vpn-server app autostart attribute *true*
- improve `--package` flag to distinguished between hypervisor and visor, and change its config output file name
- add a checking `--output` or `-o` flag, and if set was used then override `-p` and `-s` flag

How to test this PR:
- build at this PR by `make build`
- generate config with `./skywire-cli config gen --public-rpc`
- check for CLIAddr value, should be `:3435`
- add visor pk as hypervisor pks by `skywire-cli config gen --hypervisor-pks <visor-pk> -ro skywire-config.json`
- check config file that hypervisors should be empty `[]`, and hypervisor config part should added to config file.
- generate/update config with `./skywire-cli config gen -r --vpn-server-enable`
- check config file to confirm that vpn-server autostart is true
- generate config twice with `-pi` and `-p` to check their different name
- generate config with `-pir` or `-sr` with `-o config.json` flag to see override functionality
